src/components/SQForm/SQFormDatePicker.tsx

```diff
@@ -37,7 +37,7 @@ export interface SQFormDatePickerProps extends BaseFieldProps {
    */
   setDisabledDate?: (day: unknown) => boolean;
   /** Any valid prop for material ui datepicker child component - https://material-ui.com/components/pickers/  */
-  muiFieldProps?: BaseDatePickerProps<moment.Moment>;
+  muiFieldProps?: BaseDatePickerProps;
```

Comment:
I removed this because I was getting an error saying that `BaseDatePickerProps` does not accept a generic.  As you can see in the loom, the stories are still working fine.